docs/spark-connector/spark-connector.md

```diff
@@ -18,13 +18,17 @@ The Gravitino Spark connector leverages the Spark DataSourceV2 interface to faci
 
 ## Requirement
 
-* Spark 3.4
-* Scala 2.12
+* Spark 3.3 or 3.4 or 3.5
+* Scala 2.12 or 2.13
 * JDK 8,11,17
```

Comment:
`JDK 8,11,17` should be `JDK 8 or 11 or 17` ?

Reply:
done